lib/openstudio-standards/standards/Standards.WaterHeaterMixed.rb

```diff
@@ -124,21 +121,28 @@ def water_heater_mixed_apply_efficiency(water_heater_mixed)
       # Calculate the percent loss per hr
       hr_loss_base = wh_props['hourly_loss_base']
       hr_loss_allow = wh_props['hourly_loss_volume_allowance']
-      hrly_loss_pct = hr_loss_base + (hr_loss_allow / volume_gal) / 100.0
+      hrly_loss_pct = hr_loss_base + hr_loss_allow / volume_gal
       # Convert to Btu/hr, assuming:
       # Water at 120F, density = 8.25 lb/gal
       # 1 Btu to raise 1 lb of water 1 F
       # Therefore 8.25 Btu / gal of water * deg F
       # 70F delta-T between water and zone
-      hrly_loss_btu_per_hr = hrly_loss_pct * volume_gal * 8.25 * 70
+      hrly_loss_btu_per_hr = (hrly_loss_pct / 100) * volume_gal * 8.25 * 70
```

Comment:
The `/ 100` should be applied to `hrly_loss_pct`, not just to `hr_loss_allow`, otherwise `hr_loss_base` gets ignored and `hrly_loss_btu_per_hr` are overestimated.